chart/values.yaml

```diff
@@ -160,12 +162,17 @@ cronJobs: {}
 # Setting up clamav incorrectly can lead to a PRODUCTION shut down.
 clamav:
   enabled: false
+  requestedCPU: "100m"
+  requestedMemory: "3Gi"
```

Comment:
First, I like seeing sane/default values here. Thoughts on taking it a step further and using `clamav.requests.cpu` and `clamav.requests.memory` and keeping it similar in structure to the deployment manifest?

Reply:
The "requests" portion of the old values.yaml was not being used as a sub-section, but instead was being used as a yaml injection point. The anti-pattern nature of the yaml code injection from the values.yaml file was the reason for renaming these to distinct  names under the clamav grouping. Because yaml inject is such a prevalent approach, renaming gives us an inoculation against this particular behavior going forward. Modifying the environment specific values.yaml files will be a minor issue, as it involves simply removing the requests structure completely.